cli/azd/pkg/infra/provisioning/provisioning_plan.go

```diff
@@ -0,0 +1,25 @@
+package provisioning
```

Comment:
FWIW - since we also include this prefix on many types in this package, the type names "stutter" (the term of art from Go folks) when used outside the package,  eg. `provisioning.ProvisionPlan` vs just `provisioning.Plan`.  Something to keep in mind when picking names.  Not blocking feedback on any of these names, but feel free to shorten them if you'd like, it's idiomatic to consider the type name with its package.

Reply:
This makes sense since in go you typically would reference by `packageName.type` - so makes sense.